app/models/container_build.rb

```diff
@@ -0,0 +1,14 @@
+class ContainerBuild < ActiveRecord::Base
+  include CustomAttributeMixin
+  include ReportableMixin
```

Comment:
> I'm not sure we need CustomAttributeMixin, ReportableMixin, has_many :lables and acts_as_miq_taggable.
> we can take a look on Sunday. maybe @zeari knows?

@moolitayer 
```CustomAttributeMixin``` for labels\selectors - OK
```ReportableMixin``` for producing reports - should be added with a PR that adds a reports feature for builds
```acts_as_miq_taggable``` for tags - should be added with a PR that lets the user add tags to a build.

If any of the last two are coming in subsequent patch then leave them in.